apps/backend/src/modules/categories/category.controller.ts

```diff
@@ -70,6 +82,45 @@ class CategoryController extends BaseController {
 			status: HTTPCode.OK,
 		};
 	}
+
+	/**
+	 * @swagger
+	 * /categories/get-by-id:
+	 *   get:
+	 *     description: Returns an array of quiz categories
+	 *     security:
+	 *       - bearerAuth: []
+	 *     parameters:
+	 *       - in: query
+	 *         name: categoryIds
+	 *         required: true
+	 *         description: Stringified array of category ids.
+	 *         schema:
+	 *           type: string
+	 *     responses:
+	 *       200:
+	 *         description: Successful operation
+	 *         content:
+	 *           application/json:
+	 *             schema:
+	 *               type: object
+	 *               properties:
+	 *                 items:
+	 *                   type: array
+	 *                   items:
+	 *                     $ref: "#/components/schemas/Category"
+	 */
+
+	private async getCategoriesById(
```

Comment:
I think that world `Categories` is redundant in name of this method, because name of controller already have it.
```suggestion
	private async getByIds(
```